src/main/java/ch/njol/skript/config/SectionNode.java

```diff
@@ -80,6 +80,9 @@ public void add(final Node n) {
 	public void add(int index, @NotNull Node node) {
 		Preconditions.checkArgument(index >= 0 && index <= size(), "index out of bounds: %s", index);
 
+		if (node.getParent() == this) { // this is not ideal. too bad!
```

Comment:
a comment explaining _why_ (or a link to an issue) this is being done might be good